Xamarin.Essentials/Permissions/Permissions.shared.enums.cs

```diff
@@ -23,11 +23,14 @@ enum PermissionType
         Unknown,
         Battery,
         Camera,
+        CalendarRead,
+        CalendarWrite,
         Flashlight,
         LaunchApp,
         LocationWhenInUse,
         Maps,
         NetworkState,
+        Reminders,
```

Comment:
Do we need this permission?

Reply:
Nope Addressed in:
https://github.com/builttoroam/Essentials/pull/43